docs/cloud_build_standards.md

```diff
@@ -0,0 +1,90 @@
+| Project Name | Name of the project | Projectx |
+| Env Name | Name of environment for account | test, notprod, prod, ops |
+
+The overall Format would be `DIRECTORATE-COSTCODE_PORTFOLIO-PROJECT-ENV`
+
+### IP allocation
+
+If the account is going to share other services i.e. any internal central service, then the IP allocation will need to be addressed. This is to avoid IP conflicts with other services, that may also be VPN / Peered to the central internal service.
+
+Please discuss this with the team when the account is created.
+
+### General Conventions
+
+If we follow the above, then most of the environment separation will be happening at an account level rather than at a resource level. Where there are multi-tenancy type services i.e. an account with multiple service environments for applications, then the env prefix will be necessary.
+
+Please familiarise yourself with
+
+- Tagging Concepts and what services support / don't support [tagging](https://docs.aws.amazon.com/AWSEC2/latest/UserGuide/Using_Tags.html)
+- [s3 naming restrictions](https://docs.aws.amazon.com/AmazonS3/latest/dev/BucketRestrictions.html)
+
+### Tagging considerations
+
+- Tags can be applied anytime
+- Cost allocation reports are only available from the point tags were activated
+- After creating a tag [key] for costs, it must be marked activated ? added as a cost allocation tag, otherwise it will not be visible in the Cost Explorer
+
+### General Tagging Standards
+
+Below are a list of tagging standards to consider. Not all will be applicable i.e. based on the above shared account concepts (it's hosted in an account with other services), if it's clustered or not.
+
+All of the below should apply however, to all resources where you can.
+
+
+| Type | Description | Usage | Optional |
+| ---- | ------ | ------ | ------|
+| `TYPE` | Service type | CI | No |
+| `ENV` | Environment | dev, prod, uat, ppd | No |
+| `CLUSTER` | Cluster Name | cluster1, cluster2 | Yes |
+| `NODE` | Node within the cluster | node1 | Yes |
+| `APPSTAGE` | Stage of the application and resources associated | Alpha, Beta, Live | Yes |
+| `ACCESS` | Private or Public | private, public | no |
+| `PROJECT-SERVICE` | name of the service | projectx, projecty | No |
+| `PROJECT-PORTFOLIO` | Name of the portfolio | portfoliox | yes |
+| `SERVICE-TYPE` | Type of service | db, webapp, proxy | No |
+| `AZ` | availability zone number | 1a, 1b, 1c | No |
+| `COST-CODE` | cost code if a shared account | 1234-5678 | yes |
+| `REPO` | Repository related to the creation | repo1 | yes |
+| `TEAM` | Name of the team if a shared account | team1 | yes |
+
+### Additional Tagging Naming Standards
+
+| Type | Description | Usage | Optional |
+| ---- | ------ | ------ | ------|
+| `RDS-TYPE` | database type | Postgres, mysql, oracle | RDSTYPE: Oracle | No |
+
+### Automation and Cloud Rules
+
+- There should be **NO** manual changes to any resources everything should be done via code
```

Comment:
could you maybe adopt [rfc2119](https://www.ietf.org/rfc/rfc2119.txt) and change these shoulds to `MUST`s where appropriate?